sdk/lib/core/uri.dart

```diff
@@ -213,7 +253,7 @@ abstract interface class Uri {
       Map<String, dynamic /*String?|Iterable<String>*/ >? queryParameters,
       String? fragment}) = _Uri;
 
-  /// Creates a new `http` URI from authority, path and query.
+  /// Creates a new `http` URI from authority (hostname), path and query.
```

Comment:
It *can* be more than hostname. Maybe `(usually a hostname)` instead of just `(hostname)`.
Same below.